docs/superpowers/plans/2026-05-07-rob-140-research-reports-integration.md

```
@@ -0,0 +1,2404 @@
+uv run python -m scripts.ingest_research_reports \
+  --file path/to/payload.json --dry-run
+
+# Ingest
+uv run python -m scripts.ingest_research_reports --file path/to/payload.json
+```
+
+Output is a JSON summary with `inserted_count` and `skipped_count`.
+
+## Read API
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)